src/zinc/defaults.py

```diff
@@ -16,7 +16,6 @@
 :catalog_valid_formats: A list of valid formats for objects in the catalog.
 :catalog_lock_timeout: Timeout for acquiring a lock on the catalog via a coordinator.
 :catalog_prev_distro_prefix: The prefix to use when writing the previous distro.
-:storage_aws_read_retry_count: Number of times to retry an operation using an 'S3StorageBackend'.
```

Comment:
I don't believe I've ever run into the `http.client.IncompleteRead` issue that it seems like this retry count is supposed to address, so I removed the retry count feature.  I'm not sure whether it's still important or not.  I'd be happy to put it back if anyone thinks I should.  🤷‍♂️